pkg/localclusterimport/import_local_cluster.go

```diff
@@ -0,0 +1,226 @@
+package localclusterimport
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	hiveext "github.com/openshift/assisted-service/api/hiveextension/v1beta1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+	"github.com/openshift/hive/apis/hive/v1/agent"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const LocalClusterNamespace = "local-cluster"
```

Comment:
I'm not 100% sure that this is in line with the goals of the feature. The main goal I see is that we want to simplify the process of management of the hub cluster by creating some of the CRs the user needs. 

Maybe we could accept an env var optionally and default to "local-cluster"

Also note that "local-cluster" is the name of the cluster that is created in multi cluster hub.

Reply:
the only option to set env vars at the moment is with the unsupported configmap.
i don't think that it will be wise to add it as an official api at the moment, lets get a feedback on that in the next release and if needed we  will add it as an option

Reply:
but env var with a default value can be a good option even if it is set via unsupported configmap